README.md

```diff
@@ -1,2 +1,89 @@
-# final-capstone-backend
-Final capstone backend
+# Easy booking
```

Comment:
- [ ] Your API does not have any documentation.  Please add documentation about how to consume your API:

![image](https://user-images.githubusercontent.com/57421823/200048389-b9654251-91e3-4642-9cd5-c65f71a11960.png)

You could use [rswag](https://github.com/rswag/rswag) which will add the documentation and tests at the same time 💪

Reply:
Hi @Meltrust

Thank you for your feedback and your recommendations.
About the API documentation, we have used the traditional way of documenting API endpoints, you can find that in our `API.DOC.md` file in the root directory.